db/migrate/20151117231542_map_contest_group.rb

```diff
@@ -0,0 +1,11 @@
+class MapContestGroup < ActiveRecord::Migration
+    def change
+        get_all_content_groups = ContestGroup.all()
+
+        get_all_content_groups.each do |c|
+            Contest.where("name LIKE ?" , "%#{c.name}%").update_all({:contest_group_id => c.id})
+        end
+
+        Contest.where("contest_group_id IS NULL").update_all({:contest_group_id => 1})
```

Comment:
Please, don't hardcode IDs in the migration. This is fragile, as you rely to have a contest with ID 1. A better approach will be to pick a particular contest, like this:

```ruby
general_group = ContestGroup.find_by!(name: "Общи")
Contest.where("contest_group_id IS NULL").update_all(contest_group_id: general_group.id)
```